hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java

```diff
@@ -126,6 +126,9 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       omVolumeArgs.setQuotaInBytes(setVolumePropertyRequest.getQuotaInBytes());
 
+      // update modificationTime.
+      omVolumeArgs.updateModificationTime();
```

Comment:
Modification time should be set in the preExecute, and use the same value in validateAndUpdateCache. why it needs to be done like this is, in HA all OM's will have the same modification time, in this case there is a chance of OM's having different modification time.

Reply:
Thanks @bharatviswa504 for the review.
Updated with setting preExecute.